src/imageops/mod.rs

```diff
@@ -483,4 +486,18 @@ mod tests {
         let image = RgbaImage::new(50, 50);
         let _ = super::blur(&image, 0.0);
     }
+
+    #[test]
+    /// Test blur doesn't panick when passed 0.0
+    fn test_fast_blur_zero() {
+        let image = RgbaImage::new(50, 50);
+        let _ = super::fast_blur(&image, -1.0);
```

Comment:
This is incongruent with the comment, it's passed `-1.0`.

Reply:
fixed